RELEASE.md

```diff
@@ -109,3 +109,49 @@ We use [github's Releases UI](https://github.com/solana-labs/solana/releases) fo
    build` to cause a refresh of `Cargo.lock`.
 1. Push your Cargo.toml change and the autogenerated Cargo.lock changes to the
    release branch.
+
+### Update software version running on testnet.solana.com
+
+The testnet running on testnet.solana.com is set to use a fixed release tag which is set in the Buildkite testnet-management pipeline.  This tag needs to be updated and the testnet restarted after a new release tag is created.
+
+Testnet runs on the stable channel, one branch behind the newly created branch.
```

Comment:
Remove this sentence, it's not correct.  Testnet runs on a fixed release tag as described in the previous paragraph